app/views/inspections/index.html.erb

```diff
@@ -11,8 +9,6 @@
 
 <% @inspections.each do |inspection| %>
   <tr>
-    <td><%= inspection.person_id %></td>
```

Comment:
It seems like this does not leave any info about the inspection in the listing, only the buttons.